app/src/main/java/com/github/sdp/ratemyepfl/viewmodel/AddReviewViewModel.kt

```diff
@@ -67,24 +52,24 @@ class AddReviewViewModel @Inject constructor(
      *
      * @return true if it succeeds to build the review, false otherwise
      */
-    fun submitReview(): Boolean {
+    fun submitReview(item: Reviewable?): Boolean {
```

Comment:
I think this is an amazing refactor. I did not like the previous option, because we needed to refetch something we already had. I love this one.